charts/jenkins-webhook-relay/Chart.yaml

```
@@ -0,0 +1,10 @@
+apiVersion: v2
+name: recipe-receiver
+description: A Helm chart to deploy the recipe receiver app.
+type: application
+version: 0.1.0
+appVersion: "0.1.0"
+dependencies:
+  - name: functio"
+    version: "15" 
```

https://github.com/hmcts/chart-function/releases
```suggestion
    version: "2.3.4" 
```